packages/core/package-lock.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
we use yarn, so there shouldn't be a `package-lock.json` file

Reply:
keep forgetting `yarn` is the tool of choice. Will nuke it